server/utils/users.js

```diff
@@ -1,18 +1,19 @@
 const _ = require('lodash');
 
+const colors = ["#7cb5ec","#90ed7d","#f7a35c","#8085e9","#f15c80","#e4d354","#2b908f","#f45b5b","#91e8e1"];
```

Comment:
In the future, it could be added more colors to this array. The idea is that the colors should be as much different as it is possible.